### ISSUE TITLE
Ebola-specific General Condition and jump-to-form-section functionality has regressed

### ISSUE THREAD

**Issue by capnfabs** [projectbuendia/client#111]
See pull #99 for context.

This is a regression from our original deployment, which had the ability to see general condition at a glance, and jump to specific parts of the observation form for quick entry.

Filing this issue instead of tracking that loss of functionality as a comment in the code.

**Comment by capnfabs** [comment]
@dancunningham, is this something we want to track for the future or are we ok with not having this functionality any more?

**Comment by dancunningham** [comment]
Yes, we want to track this. Jump to form section is on the Roadmap, though direct obs entry will trump that, and that's how the tiles were originally designed to work.

General Condition we have to figure out.
